chapters/annotations.tex

```diff
@@ -1268,19 +1296,18 @@ \section{Graphical User Interface}\label{annotations-for-the-graphical-user-inte
 This section describes the annotations that are used to define properties of the graphical user interface.
 
 \begin{lstlisting}[language=modelica]
-annotation(preferredView = $\mathit{view}$)
+/*literal*/ constant String preferredView = $\mathit{view}$;
```

Comment:
Obsolete use of placeholder variable:
```suggestion
/*literal*/ constant String preferredView;
```
Also the default needs to be described in the test since it isn't given with a declaration equation.